x/ccv/consumer/ibc_module.go

```diff
@@ -275,15 +275,23 @@ func (am AppModule) OnTimeoutPacket(
 	packet channeltypes.Packet,
 	_ sdk.AccAddress,
 ) error {
-	var data ccv.SlashPacketData
-	if err := ccv.ModuleCdc.UnmarshalJSON(packet.GetData(), &data); err != nil {
-		return sdkerrors.Wrapf(sdkerrors.ErrUnknownRequest, "cannot unmarshal consumer packet data: %s", err.Error())
-	}
+	var slashData ccv.SlashPacketData
+	var vscMaturedData ccv.VSCMaturedPacketData
 
-	if err := am.keeper.OnTimeoutPacket(ctx, packet, data); err != nil {
-		return err
+	err := ccv.ModuleCdc.UnmarshalJSON(packet.GetData(), &slashData)
+	if err != nil {
+		vscErr := ccv.ModuleCdc.UnmarshalJSON(packet.GetData(), &vscMaturedData)
+		if vscErr != nil {
+			// packet is neither SlashPacketData nor VCSMaturedPacketData
+			return sdkerrors.Wrapf(sdkerrors.ErrUnknownRequest,
+				"cannot unmarshal consumer packet data: %s",
+				"expected SlashPacketData or VCSMaturedPacketData")
+		}
 	}
 
+	// keeper.OnTimeoutPacket is a no-op
```

Comment:
Remove `keeper.OnTimeoutPacket` as it is a noop. 

Also, add the following comment:
```
  // the CCV channel state is changed to CLOSED 
  // by the IBC module as the channel is ORDERED
```

Reply:
Done